packages/visualizations/app/styles/navi-visualizations/components/number-format-dropdown.less

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+.number-format-dropdown {
+  &__container {
+    border: 1px solid @navi-gray-300;
+    box-shadow: 0 3px 8px fade(@navi-black, 38%);
+    font-size: @font-size-base;
+    margin-top: 10px;
+    padding: 10px;
+    max-width: 280px;
+
+    &:before, &:after {
+      border: solid transparent;
+      content: " ";
+      height: 0;
+      position: absolute;
+      pointer-events: none;
+      right: 8px;
+      top: -20px;
+      width: 0;
+    }
+
+    &:before {
+      border-bottom-color: @navi-gray-300;
+      border-width: 10px;
+    }
+
+    &:after {
+      border-bottom-color: @navi-white;
+      border-width: 11px;
+    }
+
+    .number-format-selector__section-header {
+      font-family: @font-family-strong;
+      margin-bottom: 5px;
+    }
+
+    .number-format-selector__section {
+      border-bottom: 0;
+      margin-bottom: 10px;
+      padding: 0;
+    }
+
+    .number-format-selector__format-list {
+      color: @navi-gray;
+    }
+
+    .number-format-selector__format-input {
+      font-size: @font-size-base;
+      background-color: @navi-blue-100;
+      padding: 0 5px;
+      width: 100%
+    }
+
+    .number-format-selector__doc-link {
+      color: @navi-blue;
+      font-size: @font-size-mid;
```

Comment:
Can we make this a little smaller?

Reply:
![image](https://user-images.githubusercontent.com/5875315/40505511-0b0c0572-5f5a-11e8-9d59-5f090ab01f0b.png)

Reply:
done